### DISCUSSION TITLE
关于 `简化评论区` 隐藏小喇叭横幅的建议

### DISCUSSION THREAD

**Discussion by yokinanya** [the1812/Bilibili-Evolved#4868] [功能建议 / Ideas]
关于视频评论区的小喇叭横幅，是否可以通过隐藏 bili-comments-notice 来实现
![image](https://github.com/user-attachments/assets/25909da6-623a-44e9-b8aa-b04f9783ce84)
因为看到有这样子实现的脚本 https://github.com/festoney8/bilibili-cleaner

**Comment by the1812** [comment]
有 `活动横幅` 的选项, 不过最近失效了 (#4843)
